kolibri/plugins/setup_wizard/assets/src/views/LoadingTaskPage.vue

```diff
@@ -143,10 +150,11 @@
       this.pollTask();
     },
     methods: {
-      importAnother() {
-        this.isPolling = false;
-        this.wizardService.send('IMPORT_ANOTHER');
-      },
+      // To Do: role this back when multiple user syncing is working in the upcoming release!
```

Comment:
I suggest using all-caps TODO -- often they are treated as keywords in your IDE, highlighted within the comments. Also by keeping the convention of using those all-caps versions, we can reliably grep the codebase for all TODO items